command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 02:08:05PM -0700, Mike Brown wrote:
> I like that! Probably along with the proviso that the source and
> build instructions for the shim be provided for inspection or what
> not so we could check whether it is the shim that is the product
> providing compliance and not the product being tested. For, what
> would be the point / benefit of saying a shim is compliant but the
> product isn't?

I don't think we want humans reading code to be part of certification,
and am perfectly happy certifying a shim/core system as a single black
box entity [1].  Folks interested in consuming the certified runtime
should use the same shim/core system that was certfied.  Folks
interested in consuming the core directly who are suspicious of the
shim are on their own to convince themselves that the shim isn't doing
anything they need.

And if you add multiple API choices to runtime-spec, you shift the
shim into the OCI.  The the compliance team *is* responsible for
reading the code and ensuring it's not doing anything too magical.

[1]: https://github.com/opencontainers/runtime-spec/pull/513#issuecomment-231849169